desktop/apps/article/client/index.coffee

```diff
@@ -113,26 +116,23 @@ module.exports = class ArticleIndexView extends Backbone.View
 
   setupPromotedContent: =>
     if @article.get('channel_id') is sd.PC_ARTSY_CHANNEL
-      new Partner(id: @article.get('partner_ids')?[0]).fetch
-        error: => $('.articles-promoted').hide()
-        success: (partner) ->
-          new Profile(id: partner.get('default_profile_id')).fetch
-            error: => $('.articles-promoted').hide()
-            success: (profile) ->
-              $('#articles-show').prepend promotedTemplate
-                src: profile.bestAvailableImage()
-                name: partner.get('name')
-                href: profile.href()
-                type: profile.profileType()
+      send = method: 'post', query: partnerQuery(@article.get('partner_ids')?[0])
+      metaphysics send
+      .then (data) =>
+        $('#articles-show').prepend promotedTemplate
+          src: data.partner.profile.image.cropped.url
+          name: data.partner.name
+          href: data.partner.profile.href
+          type: if data.partner.type is 'Gallery' then 'Gallery' else 'Institution'
```

Comment:
I didn't see a PR on Metaphysics for this, but were you able to derive the type accurately with `data.partner.type`? What other types can this return? I'm a little worried about bundling all other types into 'Institution'.